src/logicblocks/event/processing/consumers/subscription.py

```diff
@@ -9,31 +9,67 @@
 from logicblocks.event.types import (
     Event,
     EventSourceIdentifier,
+    StoredEvent,
     str_serialisation_fallback,
 )
 
+from ...store.state import stored_event_consumer_state_converter
```

Comment:
This should be an absolute import since it crosses top-level package boundaries.